vuepress/docs/next/docs/reference/local-tips-and-tricks.md

```diff
@@ -15,14 +15,18 @@ Here are a few common questions about the quickstart environment. The quickstart
 
 **Q: What if the installation fails due to timeout?**
 
-**A:** A Docker Hub policy limiting download bandwidth may cause the quickstart installation to fail with timeout errors. The workaround is a two step process:
+**A:** A Docker Hub policy limiting download bandwidth may cause the quickstart installation to fail with timeout errors. The workaround is a two-step process. 
+
+>Note: Use the [ent CLI](../getting-started/entando-cli.md) to send commands directly to Kubernetes from the host machine.
```

Comment:
I think this version works better

Reply:
Agreed but let's remove "directly" which seems redundant. Let's also use this as a simpler version in the above cases (entando-apis, identity-managegement).